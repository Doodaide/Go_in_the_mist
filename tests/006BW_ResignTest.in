who
place A10
who
resign