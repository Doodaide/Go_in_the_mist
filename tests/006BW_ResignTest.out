B
W
Black wins!
A10
Thank you for playing!
